Docs/Configuration.tex

```diff
@@ -6964,6 +6964,13 @@ \subsection{Properties}\label{uefiprops}
   \textbf{Description}: Provide APFS support as configured in the
   \hyperref[uefiapfsprops]{APFS Properties} section below.
 
+\item
+  \texttt{AppleInput}\\
+  \textbf{Type}: \texttt{plist\ dict}\\
+  \textbf{Failsafe}: None\\
+  \textbf{Description}: Configure OpenCore's re-implementation of the Apple Event protocol
```

Comment:
`OpenCore's` is unnecessary in my opinion, as it has been pretty clear that this document is for OpenCore. Also, we had `_in_ the section` below; maybe it is a better idea to keep it for consistency.

Reply:
noted - changed to more consistent wording (I hope)